release_docs/INSTALL

```diff
@@ -0,0 +1,82 @@
+*************************************************************************
+*          Installation Instructions for HDF5                           *
+*************************************************************************
+
+Instructions for the Installation of HDF5 Software
+==================================================
+
+This file provides instructions for installing the HDF5 software.
+
+For help with installing, questions can be posted to the HDF Forum or sent to the HDF Helpdesk:
+
+   HDF Forum:     https://forum.hdfgroup.org/
+   HDF Helpdesk:  https://hdfgroup.atlassian.net/servicedesk/customer/portals
+
+CONTENTS
+--------
+        1. Obtaining HDF5
+        2. Third-party Software Requirements
+        2.1. Zlib
+        2.2  Szip (optional)
+        2.3. MPI and MPI-IO
+
+
+
+*****************************************************************************
+
+1. Obtaining HDF5
+        The latest supported public releases of HDF5 are available on
+        https://github.com/HDFGroup/hdf5.
+
+2. Third-party Software Requirements
+2.1. Zlib
+        The HDF5 library includes a predefined compression filter that
+        uses the "deflate" method for chunked datasets. If zlib-1.1.2 or
+        later is found, HDF5 will use it.  Otherwise, HDF5's predefined
+        compression method will degenerate to a no-op; the compression
+        filter will succeed but the data will not be compressed.
+
+2.2. Szip (optional)
+        The HDF5 library includes a predefined compression filter that
+        uses the extended-Rice lossless compression algorithm for chunked
+        datasets. For information on Szip compression, license terms,
+        and obtaining the Szip source code, see:
+
+           https://portal.hdfgroup.org/display/HDF5/Szip+Compression+in+HDF+Products
```

Comment:
dysfunctional link (Content not found)

Should the link be replaced or accompanied by a link for libaec:  https://gitlab.dkrz.de/k202009/libaec or https://gitlab.dkrz.de/k202009/libaec/-/releases?

Reply:
Suggestions?